### ISSUE TITLE
Can't find variable: browserMobile

### ISSUE THREAD

**Issue by leoliu** [olifolkerd/tabulator#2506]
Getting a ton of errors like this in 4.5.1:

ReferenceError
Can't find variable: browserMobile

which looks related to the following source lines:

```
js/modules/resize_table.js
13:			if(!table.browserMobile || (browserMobile &&!table.modules.edit.currentCell)){
21:			if(!table.browserMobile || (browserMobile &&!table.modules.edit.currentCell)){
```

**Comment by olifolkerd** [comment]
Hey @leoliu 

Please use the bug issue template when creating a bug report as it prompts you for the information we need to resolve the issue.

Please have a read of the [Bug Reporting Guide[(http://tabulator.info/community#bug) and resubmit this as an issue using the template.

It is essential that you provide a JS Fiddle or Code pen that demonstrates the issue you are having because it is likely only showing up in the specific configuration you are using, and without being able to replicate that we cannot easily find the route of the issue.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
In this instance i have managed to locate the issue, and pushed a fix to master which will be included in the 4.5.2 patch release.

But please make sure you follow the bug issue template in future, as the key piece of missing information needed there was that it was on a mobile web browser.

Cheers

Oli :)

**Comment by leoliu** [comment]
@olifolkerd thanks for the fix. Will try next time.
